grr/server/grr_response_server/gui/ui/components/online_chip/online_chip.ng.html

```diff
@@ -0,0 +1,6 @@
+<div *ngIf="(status$ | async) === 'online'; else offlineChip">
+  <mat-chip color="accent" selected>Online</mat-chip>
+</div>
+<ng-template #offlineChip>
+  <mat-chip>Offline</mat-chip>
```

Comment:
Please also wrap this mat-chip in a div to produce the same element hierarchy in the if and else branch. Because div is `display: block` per default, the display could change when online status changes.